themis/languages/java/installation-desktop.md

```diff
@@ -148,10 +176,3 @@ so you do not have to install Gradle manually.
 
 The resulting JAR file `java-themis-X.Y.Z.jar` will be placed into `src/wrappers/themis/java/build/libs`.
 You then need to add it to your project.
-
-To build older versions of Themis
-you need to copy the entire [`com.cossacklabs.themis` package](https://github.com/cossacklabs/themis/tree/master/src/wrappers/themis/java)
-and configure your build system to build it as a part of the application.
-
-See [JavaThemis examples](https://github.com/cossacklabs/themis-java-examples)
```

Comment:
Why removing this? I mean we could update Java examples to use JavaThemis from Gradle and suggest users to look at them

Reply:
> Why removing this?

Fat fingers :(